api/src/pcapi/core/providers/api.py

```diff
@@ -47,6 +47,18 @@ def create_venue_provider(
     else:
         new_venue_provider = connect_venue_to_provider(venue, provider, payload.venueIdAtOfferProvider)
 
+    if (
+        provider.isActive
+        and provider.enabledForPro
+        and venue.venueType.label
```

Comment:
je pense qu'il serait bon d'ajouter une condition `and not venue.isPermanent` afin d'éviter un `repository.save` inutile si le lieu est déjà marqué permanent.